modules/ezcontent_node/modules/ezcontent_smart_article/ezcontent_smart_article.module

```diff
@@ -144,9 +145,99 @@ function ezcontent_smart_article_form_node_form_alter(&$form, FormStateInterface
             $form[$term_reference_field]['widget']['smart_tags_submit']['#attributes'] = empty($summary_config->get('smart_tags_api_url')) ? ['disabled' => 'disabled'] : [];
           }
         }
+
+        // Custom submit handler for Podcast.
+        foreach ($entity->getFieldDefinitions() as $field_definition) {
+          if ($field_definition->getType() === 'ezcontent_smart_podcast') {
+            // Submit handler for podcast.
+            $form['actions']['submit']['#submit'][] = '_custom_submit_handler';
```

Comment:
Lets not generalise it. Specific name can be `ezcontent_smart_podcast_submit_handler`